contracts/CourtRoom.sol

```diff
@@ -37,7 +45,6 @@ contract CaseContract is Owned {
 		return clm.claimId;
 	}
```

Comment:
We absolutely need an id for each claim, that was not being debated. The check is intended to stop the same claim being opened twice, ie. same plaintiff and same evidence.
Can you elaborate on not needing OpenCases map? It seems an unnessacary complication for when checking and updating claims.